src/common/paths/orgs_orgID.yml

```diff
@@ -151,7 +151,7 @@ delete:
 
     #### Related guides
 
-    - [Delete organization]({{% INFLUXDB_DOCS_URL %}}/organizations/delete-orgs/)
+    - [Delete organizations]({{% INFLUXDB_DOCS_URL %}}/organizations/delete-orgs/)
```

Comment:
@sunbryely-influxdata This PR cleans up (deletes) periods for linked page titles in lists. For example...

Reply:
...also whitespace